.github/workflows/goreleaser-orbit.yaml

```diff
@@ -51,10 +51,10 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
         with:
-          go-version-file: 'go.mod'
+          go-version-file: "go.mod"
 
       - name: Run GoReleaser
-        run: go run github.com/goreleaser/goreleaser@56c9d09a1b925e2549631c6d180b0a1c2ebfac82 release --debug --rm-dist --skip-publish -f orbit/goreleaser-macos.yml # v1.20.0
+        run: go run github.com/goreleaser/goreleaser/v2@606c0e724fe9b980cd01090d08cbebff63cd0f72 release --verbose --clean --skip=publish -f orbit/goreleaser-macos.yml # v1.20.0
```

Comment:
See https://goreleaser.com/deprecations/#-debug

Reply:
You should update the version comment (`# v1.20.0`) to reflect what new version this git hash `606c0e724fe9b980cd01090d08cbebff63cd0f72` refers to.